PokeStar/PokeStar/ConnectionInterface/DatabaseConnector.cs

```diff
@@ -90,6 +90,24 @@ FROM pokemon
          return pokemon;
       }
 
+      public List<string> GetBossList(short tier)
+      {
+         List<string> bosses = new List<string>();
+         using (var conn = GetConnection())
+         {
+            conn.Open();
+            string queryString = $"SELECT pokemon FROM raid_boss WHERE tier = {tier} AND active = 1";
```

Comment:
This causes uneeded updates to the db were reading from sliph would be more efficient